include/curl/curl.h

```diff
@@ -170,7 +170,8 @@ typedef enum {
   CURLSSLBACKEND_MBEDTLS = 11,
   CURLSSLBACKEND_MESALINK               CURL_DEPRECATED(7.82.0, "") = 12,
   CURLSSLBACKEND_BEARSSL = 13,
-  CURLSSLBACKEND_RUSTLS = 14
+  CURLSSLBACKEND_RUSTLS = 14,
+  CURLSSLBACKEND_ALL = ~0
```

Comment:
I think it should be set to `-1` and be made first in the list.

Reply:
Ok, but why prefer `-1`? We want all bits set, which `~0` guarantees.

Reply:
It is not used like that. It is just supposed to be a number that does not match one of the existing (or future) TLS backends. See:

https://github.com/curl/curl/blob/3d53f211e5c907bb6e8aaddf05a3acdcae247e5d/lib/vtls/vtls.c#L1369-L1396